src/lib/repositories_v2/impl/chef_repository_impl.py

```diff
@@ -14,11 +14,12 @@ def __init__(self):
         self.session = ioc.get_instance("sqlalchemy_session")
 
     def add(self, chef):
-        chef.created_date = datetime.now()
-        chef.updated_by = chef.created_by
-        chef.updated_date = chef.created_date
-        self.session.add(chef)
-        self.session.commit()
+
+        with self.session.begin():
```

Comment:
Could you move this to the beginning of the function?

Reply:
Do it for all other cases

Reply:
done.